test/normal/toerror_test.go

```diff
@@ -0,0 +1,79 @@
+//  Copyright 2019 Ingun Jon
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+type LocalType struct{}
+
+func true0() bool {
+	return true
+}
+func false0() bool {
+	return false
+}
+func true1() (int, bool) {
+	return 0, true
+}
+func true2(a int) (int, bool) {
+	return a, true
+}
+func true3(a, b int) (int, bool) {
+	return a + b, true
+}
+func true4(a, b int) (int, int, bool) {
+	return a, b, true
+}
+func true5(lt *LocalType) (*LocalType, bool) {
+	return lt, true
+}
+func true6(t *time.Time) (*time.Time, bool) {
+	return t, true
+}
+
+// func fail
```

Comment:
I don't know if the comment is necessary, but otherwise this test looks much cleaner :)